### ISSUE TITLE
TypeError: Cannot read property 'properties' of null

### ISSUE THREAD

**Issue by Beluk** [lelouch77/docusaurus-lunr-search#25]
When building with 'npm run build' i got the following error. Any help would be greatly appreciated.

![image](https://user-images.githubusercontent.com/10911904/95251695-34085580-081c-11eb-89a6-c21b658e7461.png)

**Comment by flavius-tirnacop-hs** [comment]
Adding a bit more info to this. It looks that it happens for h2/h3 like `## Jest architectural overview` 
```js
{
  type: 'element',
  tagName: 'h2',
  properties: {},
  children: [
    {
      type: 'element',
      tagName: 'a',
      properties: [Object],
      children: [],
      position: [Object]
    },
    {
      type: 'text',
      value: 'Jest architectural overview',
      position: [Object]
    },
    {
      type: 'element',
      tagName: 'a',
      properties: [Object],
      children: [Array],
      position: [Object]
    }
  ],
  position: {
    start: { line: 55, column: 1246, offset: 139174 },
    end: { line: 55, column: 1516, offset: 139444 }
  }
}
```

Reverted to v2.1.0 and everything works.

**Comment by Beluk** [comment]
Thank you, i tested v2.1.0 and it is working fine.

**Comment by lelouch77** [comment]
Issue fixed as part of the PR #28,  new version published 2.1.8.
Thanks to @zmogas
